### PR TITLE
Add Symfony Live London

### PR DESCRIPTION
**Pull request by krzystof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@
 
  - (September 9 - 10) [RustConf](http://rustconf.com/) (Portland, OR) - A day of trainings and a day of talks on the Rust programming language
  - (September 12 - 18) [Techweek: Kansas City](http://techweek.com/kansascity/) - The nation's leading technology conference and festival - showcasing entrepreneurship and innovation.
+ - :star: (September 15 - 16) [SymfonyLive](http://london2016.live.symfony.com/) (London, UK) - best practices, experience, knowledge and the latest developments with the framework
```
Comment (mAiNiNfEcTiOn) [inline README.md:48]: Why a :star:? Any proper reason to be distinguished from others?

Comment (trstringer) [inline README.md:48]: @mAiNiNfEcTiOn that's how consumers know that it is a newly added list.  See the contribution guidelines on the readme, including the :star: is the right format. Thanks!

**Comment by trstringer** [comment]
@krzystof Thanks for the contribution!  It looks like there are merge conflicts, can you resolve these conflicts?  I'll merge the PR then.  Thanks! 👍 😄

**Comment by krzystof** [comment]
@tstringer fixed the conflicts, good to go!

**Comment by trstringer** [comment]
@krzystof perfect, looks great!  Thanks! 👍 😄

File: README.md (hunk 1 above)
Comment (mAiNiNfEcTiOn) [inline README.md:48]: @tstringer oh, ok then ;-) Thanks for pointing that out.

**Pull request merged**